crates/client/src/client.rs

```diff
@@ -750,10 +770,10 @@ impl FuelClient {
     // Retrieve a page of balances by their owner
     pub async fn balances(
         &self,
-        owner: &str,
+        owner: &Address,
         request: PaginationRequest<String>,
```

Comment:
Hmm, maybe it also would be nice to use a real cursor instead of the `String`. If that possible, of course

Reply:
Updated

Reply:
As discussed over a call, we can investigate this further at a separate time. I am creating this ticket to capture this tech debt: https://github.com/FuelLabs/fuel-core/issues/1197